Sources/PackageRegistryTool/PackageRegistryTool+Publish.swift

```diff
@@ -148,15 +163,15 @@ extension SwiftPackageRegistryTool {
                 packageIdentity: self.packageIdentity,
                 packageVersion: self.packageVersion,
                 packageDirectory: packageDirectory,
-                metadataPath: publishRequirements.metadata.location.contains(.archive) ? metadataPath : .none,
-                customWorkingDirectory: self.customWorkingDirectory,
+                metadataPath: publishConfiguration.metadata.location.contains(.archive) ? metadataPath : .none,
```

Comment:
There is the possibility of `customMetadataPath` being used here then?

Reply:
this behavior has not been updated yet to the logic mentioned above, it is based on the PublishConfiguration (which used to be set by the server)